hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/AbstractAutoCreationLogic.java

```diff
@@ -211,6 +213,17 @@ public Pair<ResponseCodeEnum, Long> create(
         sideEffects.trackAutoCreation(newId);
 
         final var childRecord = creator.createSuccessfulSyntheticRecord(NO_CUSTOM_FEES, sideEffects, memo);
+
+        if (!isAliasEVMAddress) {
+            final var key = asPrimitiveKeyUnchecked(alias);
+
+            if (key.hasECDSASecp256K1()) {
+                final JKey jKey = asFcKeyUnchecked(key);
+                final var evmAddress = tryAddressRecovery(jKey, EthSigsUtils::recoverAddressFromPubKey);
```

Comment:
`tryAddressRecovery()` can in principle return `null` (although it looks like that case would represent a failure of upstream logic?)

But we should prob handle `null` here.

Reply:
Yup if there is something wrong in `tryAddressRecovery()`, WARN will be logged. 
Added `null` check, however SonarCloud is now failing with coverage. 
To fix  that I need to pass correct ECDSA key, but the recovered EVM from that key needs to be `null` and that state cannot be reached ( unless I mock the `AliasManager.tryAdressRecovery()` to return `null` ) But is it necessary ?

Reply:
Yes, the upstream [AliasResolver checks will have already caught an invalid key alias](https://github.com/hashgraph/hedera-services/blob/2796e5ac53d2156d4d8d17954e4919807fbd9853/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/grpc/marshalling/AliasResolver.java#L246-L254), and we wouldn't have reached the `TransferLogic` execution. If we are executing this code here, we can be fairly certain the alias is a valid one. 

Otherwise, the [alias -> Key --> JKey conversions from a couple of lines above](https://github.com/hashgraph/hedera-services/blob/ddeba215f2232985225453a417ec6061e8602e38/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/AbstractAutoCreationLogic.java#L185-L186) could also, in theory, throw if the alias is invalid, but we do not handle such cases either... Personally, I think we can omit the `null` check here 😶‍🌫️